homeassistant/components/intellifire/translations/en.json

```diff
@@ -7,6 +7,9 @@
         "cannot_connect": "Could not connect to a fireplace endpoint at url: http://{host}/poll\nVerify IP address and try again"
       },
       "step": {
+        "dhcp_confirm": {
```

Comment:
Don't forget to run `python3 -m script.translations develop` to update this file